Documentation/ceph-upgrade.md

```diff
@@ -138,11 +138,11 @@ Finally, if you desire to use the beta snapshots, check that the [prerequisites]
 ## Patch Release Upgrades
 
 Unless otherwise noted due to extenuating requirements, upgrades from one patch release of Rook to
-another are as simple as updating the image of the Rook operator. For example, when Rook v1.4.8 is
-released, the process of updating from v1.4.8 is as simple as running the following:
+another are as simple as updating the image of the Rook operator. For example, when Rook v1.4.9 is
+released, the process of updating from v1.4.9 is as simple as running the following:
```

Comment:
Leave as 1.4.8 I think?

Reply:
good catch

Reply:
I.e., it should say "when Rook v1.4.9 is released, the process of updating from v1.4.8 is as ...."